source/libs/page-detect.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable unicorn/prefer-starts-ends-with */
+/* eslint-disable unicorn/prefer-starts-ends-with, @typescript-eslint/prefer-string-starts-ends-with */
```

Comment:
@bfred-it I didn't want to touch this as we already ignored a similar rule here, but I think we should fix this. Relying on `RegExp#test` coercing `undefined` to a string can be dangerous, as `undefined` might actually match something, causing a false-positive.

Reply:
Ugh, you're right. `/undefined/.test(undefined) => true`